src/api/assets-api.ts

```diff
@@ -0,0 +1,16 @@
+import { getConfig, AssetsApi, ListAssetsResponse, EthNetwork } from "@imtbl/core-sdk";
+
+// Inspired by this: https://stackoverflow.com/questions/37978528/typescript-type-string-is-not-assignable-to-type
+const ethNetwork = process.env.REACT_APP_ETHNETWORK as EthNetwork
```

Comment:
it would be nice to add the `env.example` file or describe the ENV variable in the README file.